lib/client.js

```diff
@@ -149,7 +149,7 @@ const requestWithFilters = (params, filters) => {
     .then(
         (rawResponse) => {
             response = rawResponse;
-            return decodeResponse(rawResponse);
+            return decodeResponse(rawResponse, params.headers.accept);
```

Comment:
`headers` is an optional property, as is `accept`, so you should still provide a fallback.